src/main/java/org/takes/http/BkTimeable.java

```diff
@@ -130,7 +61,56 @@ private void check() {
      * @param msec Execution latency
      */
     BkTimeable(final Back back, final long msec) {
-        super(new BkThreads(msec, back));
+        super();
+        this.threads = new ConcurrentHashMap<Thread, Long>(1);
+        this.back = back;
+        this.latency = msec;
+    }
+
+    /**
+     * This method will start the damon thread.
+     */
+    public void startDaemonThread() {
```

Comment:
@sebing oops, that's bad... all public methods must be "Override"... if you have a method that is public and is not implementing any interface - the design is not good. think again.